app/Http/Controllers/Api/V1/ListSpeedtestServers.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers\Api\V1;
+
+use App\Actions\GetOoklaSpeedtestServers;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use OpenApi\Attributes as OA;
+
+class ListSpeedtestServers extends ApiController
+{
+    #[OA\Get(
+        path: '/api/v1/speedtests/servers',
+        description: 'Get a list of available Ookla speedtest servers.',
+        responses: [
+            new OA\Response(response: Response::HTTP_OK, description: 'OK'),
+            new OA\Response(response: Response::HTTP_FORBIDDEN, description: 'Forbidden'),
+        ]
+    )]
+    public function __invoke(Request $request)
+    {
+        if ($request->user()->tokenCant('speedtests:run')) {
```

Comment:
Let's create a new token ability called "ookla:list-servers" so we can be granular with scoping abilities. 

You'll also need to add it to `app/Filament/Pages/ApiTokens.php` page in the admin panel.